examples/commonjs/src/index.ts

```diff
@@ -0,0 +1,60 @@
+const {
+  OperationType,
+  Snitch,
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+} = require("@streamdal/snitch-node-client/snitch");
+
+const exampleData = {
+  boolean_t: true,
+  boolean_f: false,
+  object: {
+    ipv4_address: "127.0.0.1",
+    ipv6_address: "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+    mac_address: "00-B0-D0-63-C2-26",
+    uuid_dash: "550e8400-e29b-41d4-a716-446655440000",
+    uuid_colon: "550e8400:e29b:41d4:a716:446655440000",
+    uuid_stripped: "550e8400e29b41d4a716446655440000",
+    number_as_string: "1234",
+    field: "value",
+    empty_string: "",
+    null_field: null,
+    empty_array: [],
+  },
+  array: ["value1", "value2"],
+  number_int: 100,
+  number_float: 100.1,
+  timestamp_unix_str: "1614556800",
+  timestamp_unix_num: 1614556800,
+  timestamp_unix_nano_str: "1614556800000000000",
+  timestamp_unix_nano_num: 1614556800000000000,
+  timestamp_rfc3339: "2023-06-29T12:34:56Z",
+};
+
+const config = {
+  snitchUrl: "localhost:9091",
+  snitchToken: "1234",
+  serviceName: "test-service-name",
+  pipelineTimeout: "100",
+  stepTimeout: "10",
+  dryRun: false,
+};
+
+const audience = {
+  serviceName: "test-service-name",
```

Comment:
Yeah I was planning on doing that in a follow up since this is such a large infrastructural change...so that I wouldn't lose any unrelated changes if we have to revert.

Reply:
This will take a little bit more work...I've ticketed it for follow up.